requirements.txt

```diff
@@ -1,3 +1,3 @@
-OWSLib==0.9.2
+OWSLib==0.18.0
```

Comment:
What are these requirements about?

Reply:
The thing is that I don't remember why that was necessary. Either the build or my tests failed because I was on a lower version of owslib. Next time I will come up with a better commit message that I will know at the time that I notice that the upgrade is needed. It is a good point though and I wish I would have documented it at the time it came up.

Reply:
It was failing for you locally because you were still using the old Docker container with an outdated certificate. We tried increasing the version requirement of OWSLib to see if that would fix it (before we realised the problem was the container). I would keep the requirement like this rather than stay with the lower version, since everything is working this way.